samples/gen-go-db/server/db/dynamodb/dynamodb-local.sh

```diff
@@ -18,4 +18,4 @@ then
     tar -zxvf /tmp/dynamodb_local_latest.tar.gz -C /tmp/
 fi
 
-exec java -jar "$jar" -sharedDb -inMemory -port 8002
+exec java -jar "$jar" -sharedDb -port 8002
```

Comment:
what's the reason for this change? I don't remember why i chose `inMemory` originally

Reply:
if the db crashed after tables are created but before queries are sent to the table, the db will be restarted without the tables because they were stored in memory. Removing this option writes the db to a file so it persists across crashes.